### PR TITLE
Enable driver installation on MacOS

### PR DESCRIPTION
**Pull request by samuelpowell**

### CODE REVIEW

**Comment by samuelpowell** [comment]
@ReubenHill passes CI, okay for you locally?

File: deps/build.jl [hunk 1]
```diff
@@ -14,17 +16,47 @@ function validate_libFTD2XX_version(name, handle)
     v = Ref{Cuint}()
     s = ccall(f, Culong, (Ref{Cuint},), v)
     s == C_NULL && return false
-    return v[] >= 0x00021228
+    if Compat.Sys.iswindows()
+        return v[] >= 0x00021228
```
Comment (ReubenHill) [inline deps/build.jl:20]: What does this do?

File: deps/build.jl [hunk 2]
```diff
@@ -14,17 +16,47 @@ function validate_libFTD2XX_version(name, handle)
...
+    else
+        return true         # OS X library returns version 0x0000000
+    end
 end
 
-libFTD2XX = library_dependency("libFTD2XX", aliases = ["ftd2xx64", "ftd2xx", "libftd2xx"], validate=validate_libFTD2XX_version)
+libFTD2XX = library_dependency("libFTD2XX", aliases = ["ftd2xx64", "ftd2xx", "libftd2xx", "libftd2xx.1.4.4"], validate=validate_libFTD2XX_version)
```
Comment (ReubenHill) [inline deps/build.jl:26]: Why the change here?

File: deps/build.jl [hunk 3]
```diff
@@ -6,6 +6,8 @@ using BinDeps
 
 Sys.WORD_SIZE != 64 && error("Build script configured only for x64")
 
+download_dir = joinpath(@__DIR__, "downloads")
```
Comment (ReubenHill) [inline deps/build.jl:9]: Only used for OSX build. Maybe move there? Also are you sure you'll have `@__DIR__` pointing somewhere sensible when you probe it?

**Review by ReubenHill — APPROVED** [review]
Works fine on windows. See comments for my questions and my point about the `download_dir`. Assuming this runs fine on MacOS I'm happy for this to be merged to master.

File: deps/build.jl (hunk 2 above)
Comment (samuelpowell) [reply deps/build.jl:26]: OS X lib name is different from windows

File: deps/build.jl (hunk 1 above)
Comment (samuelpowell) [reply deps/build.jl:20]: Only windows has a version number function, so we check it, but we don't on OS X
